src/rc_client.c

```diff
@@ -3247,6 +3247,275 @@ const rc_client_subset_t* rc_client_get_subset_info(rc_client_t* client, uint32_
   return NULL;
 }
 
+/* ===== Game Info ===== */
+
+typedef struct rc_client_fetch_hash_library_callback_data_t
+{
+  rc_client_t* client;
+  rc_client_fetch_hash_library_callback_t callback;
+  void* callback_userdata;
+  uint32_t console_id;
+  rc_client_async_handle_t async_handle;
+} rc_client_fetch_hash_library_callback_data_t;
+
+static void rc_client_fetch_hash_library_callback(const rc_api_server_response_t* server_response, void* callback_data)
+{
+  rc_client_fetch_hash_library_callback_data_t* hashlib_callback_data =
+    (rc_client_fetch_hash_library_callback_data_t*)callback_data;
+  rc_client_t* client = hashlib_callback_data->client;
+  rc_api_fetch_hash_library_response_t hashlib_response;
+  const char* error_message;
+  int result;
+
+  result = rc_client_end_async(client, &hashlib_callback_data->async_handle);
+  if (result)
+  {
```

Comment:
Opening braces should not be on a separate line (except for the opening brace of the function - which isn't consistent between files). 

I wonder if there's something wrong with the `.editorconfig`, which does say `curly_bracket_next_line  = false`, but I often have to manually correct my files too. Maybe adding an `indent_brace_style` would help. I'll play around with this separately.

Reply:
It's probably because I was editing it within the DS tree, and it was picking up my `clang-format` in the root there. So likely no issue on your end.